docs/source/configuration.rst

```diff
@@ -131,4 +131,5 @@ landmarks_sampling_ratio
 .. _sensor_msgs/PointCloud2: http://docs.ros.org/api/sensor_msgs/html/msg/PointCloud2.html
 .. _sensor_msgs/NavSatFix: http://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html
 .. _cartographer_ros_msgs/LandmarkList: https://github.com/googlecartographer/cartographer_ros/blob/4b39ee68c7a4d518bf8d01a509331e2bc1f514a0/cartographer_ros_msgs/msg/LandmarkList.msg
+.. _cartographer_ros_msgs/LandmarkEntry: https://github.com/googlecartographer/cartographer_ros/blob/4b39ee68c7a4d518bf8d01a509331e2bc1f514a0/cartographer_ros_msgs/msg/LandmarkEntry.msg
```

Comment:
general remark: isn't it better to use a reference to master instead to a specific commit? i.e. replace "blob/<git hash>/" by "blob/master"

I'm not sure but wanted to point that out :)

Reply:
I just copied the link from LandmarkList. So I have no idea. I mean the blob/master could change? Maybe?

Reply:
> I mean the blob/master could change?
Yes, the advantage would be that it doesn't go unnoticed if the documentation is outdated or the message file changes. But there could be also an advantage with fixing it to a commit, not 100% sure

Reply:
You know what, I go with your idea. It is better to change all on the current master. How do I apply these changes from the GitHub WebInterface? I never used it and I do not have my normal workstation here? Used to GitLab.

Reply:
Never mind, figured it out.